native-hello-module/README.md

```diff
@@ -1,5 +1,16 @@
-# GraalVM Demos: Building a HelloWorld Java Module into a Native Executable
-This repository contains the code for a demo application for [GraalVM](http://graalvm.org).
+# Build Java Module into a Native Executable
+
+GraalVM Native Image supports the Java Platform Module System, introduced in Java 9, which means you can convert a modularized Java application into a native executable.
+
+The native-image tool accepts the module-related arguments like `--module (-m)`, `--module-path (-p)`, `--add-opens`, `--add-exports` (same as for the java launcher). When such a module-related argument is used, the native-image tool itself is used as a module too.
+
+In addition to supporting `--add-reads` and `--add-modules`, all module related options are considered prior to scanning the modulepath. This helps prevent class loading errors and allow for better module introspection at run time.
+
+The command to build a native executable from a Java module is:
+```bush
+native-image [options] --module <module>[/<mainclass>] [options]
+```
+The example below shows how to build a modular Java application into a native executable. 
 
 ## Prerequisites
```

Comment:
@LesiaChaban, please remove the Prerequisites part completely. It is self-explanatory that a user must have GraalVM installed if he is in the GraalVM Demos repository. You also tell how to install GraalVM with NI in the `#Preparation` section.